### PR TITLE
[EN] Ranking_Criteria/Skin_Set_List/ - ubkrc_skinning RC proposal table

### PR DESCRIPTION
**Pull request by Okorin**
This contains the table with all skinsets that are relevant to the new Ranking Criteria proposal which will hopefully be posted on Sunday, a html table was chosen because markup tables do not support colspan and rowspan and making multiple tables just makes everything way more complicated than necessary (splitting this up would create 50 times more redundancy and make this already long as heck table even more long as heck). 

This choice was made to avoid cluttering the page with redundant information, I would be able to switch and split this up into multiple tables, but would lose but would lose a lot of its compact information style.

### CODE REVIEW

File: wiki/Ranking_Criteria/skinning/en.md [hunk 1]
```diff
@@ -0,0 +1,1002 @@
...
+	</tr>
+	<tr>
+		<td>followpoint.png</td>
+		<td>-</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>hitcircle.png</td>
+		<td dir="ltr" rowspan="2">should always be inlcuded together, used for slidercircles when they are not skinned, won't be used for slidercircles if an user-specific skin has them skinned</td>
```
Comment (oralekin) [inline wiki/Ranking_Criteria/skinning/en.md]: `a user-specific`,  the sound isn't open

File: wiki/Ranking_Criteria/skinning/en.md [hunk 2]
```diff
@@ -0,0 +1,1002 @@
...
+	</tr>
+	<tr>
+		<td>reversearrow.png</td>
+		<td>should face right</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<td>sliderendcircle.png</td>
+		<td dir="ltr" rowspan="4">should be included to guarantee correct display when an user-specific skin has them skinned</td>
```
Comment (oralekin) [inline wiki/Ranking_Criteria/skinning/en.md]: `a`, see previous
There are more of these btw, not going to go through them all.

File: wiki/Ranking_Criteria/skinning/en.md [hunk 3]
```diff
@@ -0,0 +1,1002 @@
...
+	<tr>
+		<td>default-9.png</td>
+		<td dir="ltr">required</td>
+	</tr>
+	<tr>
+		<th rowspan="13">osu!<br>
+		Spinner Set</td>
+		<td>spinner-background.png</td>
+		<td dir="ltr" rowspan="3">basic spinner style,<br>
+		will be prioritized if both styles are present<br>
```
Comment (oralekin) [inline wiki/Ranking_Criteria/skinning/en.md]: maybe use a period, the break isn't very obvious

File: wiki/Ranking_Criteria/skinning/en.md [hunk 4]
```diff
@@ -0,0 +1,1002 @@
...
+	</tr>
+	<tr>
+		<td>spinner-approachcircle.png</td>
+		<td>used by both sets</td>
+		<td dir="ltr">required</td>
+		<td dir="ltr">384x384</td>
+	</tr>
+	<tr>
+		<td>spinner-clear.png</td>
+		<th rowspan="2">should always be included together</td>
```
Comment (oralekin) [inline wiki/Ranking_Criteria/skinning/en.md]: This has a header tag (`<th...>`) for no reason(?).

File: wiki/Ranking_Criteria/skinning/en.md [hunk 5]
```diff
@@ -0,0 +1,1002 @@
...
+		<td>selection-mod-suddendeath.png</td>
+		<td dir="ltr">required</td>
+		<td>-</td>
+	</tr>
+	<tr>
+		<th dir="ltr" rowspan="9">Playfield Set</td>
+		<td>comboburst-fruits(-n).png</td>
+		<td>should face right, automatically flipped if shown on the right side</td>
+		<td>optional</td>
+		<th rowspan="3">max height: 768px</td>
```
Comment (oralekin) [inline wiki/Ranking_Criteria/skinning/en.md]: This has a header tag for no reason(?) too.

**Comment by Okorin** [comment]
changed all of those i think

File: wiki/Ranking_Criteria/skinning/en.md [hunk 6]
```diff
@@ -0,0 +1,1002 @@
+Skin Set List 
+============= 
+
+The following table contains all skin sets that are presented in the draft worked out by the skinning subdivision of the [UBKRC-Project](https://osu.ppy.sh/forum/t/480331) (name credits go to Ephemeral)
+
+<table cellpadding="0" cellspacing="0">
```
Comment (kj415j45) [inline wiki/Ranking_Criteria/skinning/en.md]: if you can, use `Markdown table` please...

Comment (Okorin) [inline wiki/Ranking_Criteria/skinning/en.md]: I can, but that would blow the entire listing out of proportion by a big shot and decrease readability and usability by adding a lot of redundant information, but it's possible

**Comment by MegaApplePi** [comment]
Please change the title of this pr to something like `[EN] Ranking_Criteria/skinning` the current title is vague.

**Comment by Okorin** [comment]
changed title, translated the entire thing into a ton of GFM tables

File: wiki/Ranking_Criteria/skinning/en.md [hunk 7]
```diff
@@ -0,0 +1,331 @@
+Skin Set List
```
Comment (TPGPL) [inline wiki/Ranking_Criteria/skinning/en.md]: All `-` in tables into `N/A`?

File: wiki/Ranking_Criteria/skinning/en.md [hunk 8]
```diff
@@ -0,0 +1,331 @@
...
+|fruit-orange-overlay.png   |-   | required  |128x128   |
+|fruit-bananas.png   |should be designed differently from the other fruits   |required   |128x128   |
+|fruit-bananas-overlay.png   |should be designed differently from the other fruits   | required  |128x128   |
+|fruit-drop.png   |-   |required   |82x103   |
+|fruit-drop-overlay.png   |should be included to guarantee correct display when an user-specific skin has it skinned because the default skin uses a transparent image for this file   |required   |82x103   |
+
+## osu!catch Catcher Set
+
+All elements besides lightning.png are gameplay relevant. 
+For a catcher the plate must be perfectly centered and exactly 302px across in width.
```
Comment (oralekin) [inline wiki/Ranking_Criteria/skinning/en.md]: `For catchers, the plate`

File: wiki/Ranking_Criteria/Skinning/en.md [hunk 9]
```diff
@@ -0,0 +1,331 @@
...
+|--:|:-:|:-:|:-:|
+|hit0(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|hit50(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|hit100(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|hit100k(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|hit300(-n).png   | should be clearly distinguishable from all other hitbursts   | required  | -  |
+|hit300g(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|hit300k(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|lighting.png   | -  | optional  | -  |
+|particle50.png   | should be included together with all other particles; will only work if custom hitbursts are included   | optional  | 7x7  |
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/Skinning/en.md]: I don't know how repetitive you want to be, but most of the dimensions listed don't contain units.
e.g. `11x17` should be `11x17px` 🤔

File: wiki/Ranking_Criteria/Skinning/en.md [hunk 10]
```diff
@@ -0,0 +1,331 @@
+Skin Set List 
+============= 
+
+The following table contains all skin sets that are presented in the draft worked out by the skinning subdivision of the [UBKRC-Project](https://osu.ppy.sh/forum/t/480331) (name credits go to Ephemeral)
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/Skinning/en.md]: add a period `.` to the end of this sentence (which ironically this one doesn't \:p )

**Comment by Okorin** [comment]
using N/A will just confuse people because we defined "undefined" as "doesn't matter" on draft

i think renaming folder worked
maybe

File: wiki/Ranking_Criteria/Skinning/en.md [hunk 11]
```diff
@@ -0,0 +1,331 @@
...
+
+## osu! Cursor Set
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/Skinning/en.md]: Just to ask, does the headings with prefixing game modes need to to prefix the game mode?
If so, I suggest making the game mode a level 2 followed by the set which would be a level 3 heading

```md
## osu!

### Cursor Set

[...]

## osu!taiko

### Hitobject Set

[...]

```

File: wiki/Ranking_Criteria/Skinning/en.md (hunk 9 above)
Comment (oralekin) [inline wiki/Ranking_Criteria/Skinning/en.md]: Declaring that all sizes are in pixels at the start would be better?

**Comment by Okorin** [comment]
i just went for doing the whole thing because it seems neater

File: wiki/Ranking_Criteria/Skinning/en.md [hunk 12]
```diff
@@ -0,0 +1,339 @@
+Skin Set List
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/Skinning/en.md]: either, change this to say the folder's name or change the folder's name to match this

**Comment by Okorin** [comment]
¯\_(ツ)_/¯

**Pull request merged**